scripts/src/ui/page/multichoice.question.tsx

```diff
@@ -1,29 +1,53 @@
 import * as React from "react";
 import BasePage from "./base";
 import "../../../../styles/src/page/multichoice.question.styl";
+import { ButtonHTMLAttributes } from "react";
 
 export interface FullPageMultiChoiceProps {
 	id: string;
 	title: string;
+	description: string;
 	choices: string[];
-	onSelected?: (answerData: {}) => void;
+	onSelected: (answerData: {}) => void;
```

Comment:
don't use `{}` as a type, it will accept EVERYTHING:
```
let a: {};

a = 5; // ok
a = true; // ok
a = "string"; // ok
a = [1, 2, 3]; // ok
a = { key: "value" }; // ok
```

If you really want to accept anything, then use the type `any`.
If you want to accept an object then use the type `object`:

```
let a: object;

a = 5; // error
a = true; // error
a = "string"; // error
a = [1, 2, 3]; // ok
a = { key: "value" }; // ok
```

Or, you can use an indexable type for only object literals: `{ [key: string]: any }`